ts/src/test/tests.ts

```diff
@@ -1114,7 +1114,7 @@ class testMainClass extends baseMainTestClass {
         return newInput;
     }
 
-    async testMethodStatically (exchange, method: string, data: object, type: string, skipKeys: string[]) {
```

Comment:
two things here - first: we had `test Response Statically` method, which is used inside `testExchangeResponseStatically` method. then we need also this to be named `testRequestStatically` because this method is used in `testExchangeRequestStatically`